container/controller.go

```diff
@@ -621,6 +625,9 @@ func (c *Controller) Run() (err error) {
 	endpointExit := make(chan struct{})
 	c.endpoints.Run(endpointExit)
 
+	// Start CC Rest API Proxy
+	go c.ccApiProxy.run() // TODO how to control shutdown?
```

Comment:
we could use the `context` package here to do it, but I think don't worry about it. We'll need to implement graceful shutdown from a channel across the board later when we do wire in context.